keyboards/nifty_numpad/keymaps/idle_rgb_example/keymap.c

```diff
@@ -0,0 +1,179 @@
+// Copyright 2023 Isaac Rex (@Acliad)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+#include "nifty_numpad.h"
+#include "quantum.h"
```

Comment:
Shouldn't be needed.  `QMK_KEYBOARD_H` should resolve to `nifty_numpad.h` for this keyboard, and that file should have quantum.h included.
```suggestion
```

Reply:
Updated